charts/posthog/templates/ingress.yaml

```diff
@@ -94,12 +94,25 @@ spec:
             path: "/capture"
             backend: *INGESTION
 
+          {{- if .Values.decide.enabled}}
```

Comment:
When we enable this, will Helm wait for the decide deployment to be ready before updating the ingress?

I have a similar PR [here](https://github.com/PostHog/charts-clickhouse/pull/666) but wasn't sure what the behaviour was.

Reply:
ohhh awesome I was going to follow up with this too hahaha

Yeah that is a little worrying. In that case I might adjust this so that we can toggle the ingress separately